docs/core-concepts/versioning.md

```diff
@@ -1,6 +1,201 @@
+
+### netstandard
+
+.NET Standard is a new concept that came out of the .NET Core project, but is logically separate. It is a a new framework that describes the API evolution of .NET for the lower-most APIs. One could consider it a standardized Base Class Library (BCL) for all .NET implementations. .NET Core implements this spec and is ".NET Standard compliant" at a particular version level. The .NET Standard will be updated regularly, based on new APIs being adding in .NET Standard compliant .NET implementations, such as .NET Core, .NET Framework and Mono. The version number of .NET Standard will not match any of the implementations.
+
+The TFM for .NET Standard is `netstandard`. At the release of .NET Core 1.0, there will be several `netstandard` versions, not just a 1.0 version. There was a strong desire to create a bridge between Portable Class Libraries (PCL) and .NET Standard.  To establish the compatibility bridge, .NET Standard was pre-versioned to overlap with existing .NET platforms in a compatible way with PCL. .NET Standard versions 1.0 through 1.4 represent the overlap. The .NET Core 1.0 platform implements all .NET Standard versions up to and including 1.5.
+
+PCL and .NET Standard both enable binary sharing but define the applicable APIs in different ways. .NET Standard is the new binary code sharing mechanism going forward.
+
+### netcoreapp
+
+The `netcoreapp` framework represents the APIs available in the .NET Core distribution, which you can get via the .NET Core installer or a package manager. It matches the Microsoft.NETCore.App meta-package. .NET Core apps should use this framework, as should libraries that require significant funtionality beyond .NET Standard.
+
+Other .NET platforms could support this framework, but are not expected to.
+
+.NET Core and .NET Standard Versioning
+======================================
+
+The .NET Standard will not adopt the version numbers of .NET Core. They are logically separate. There are three major scenarios to consider.
+
+The following sections discuss API additions. It assumes that the additions are to libraries that are referenced by NETStandard.Library meta-package.
+
+Adding APIs to .NET Core
+------------------------
+
+When APIs are added to .NET Core, they will also be added .NET Standard. Both .NET Core and .NET Standard will version. The APIs might be added to other .NET implementations at the same time, such as the .NET Framework. 
+
+Adding APIs to another .NET Implementation
+------------------------------------------
+
+When APIs are added to the .NET Framework (for example), but not .NET Core, the .NET Standard will version, but .NET Core will not.
+
+The next time .NET Core ships, it will (one assumes) add those new APIs such that it now supports that higher .NET Standard version. It can also add additional APIs at the same time, which could create a new .NET Standard version.
```

Comment:
Hmmm...maye change the "one assumes" to something that is stronger. No-one assumes, we *know* 'cause we're building the product, correct? Something like "The next time .NET Core ships, it may add those new APIs if the collaborators decide it makes sense; this will in turn mean that .NET Core supports that higher .NET Standard version". Or similar.